pkg/plugins/generic/generic_plugin.go

```diff
@@ -81,7 +81,8 @@ type genericPluginOptions struct {
 	skipBridgeConfiguration bool
 }
 
-const scriptsPath = "bindata/scripts/kargs.sh"
+const daemonScriptsPath = "bindata/scripts/kargs.sh"
+const systemdScriptsPath = "//var/lib/sriov/kargs.sh"
```

Comment:
nit. remove double '/'
```suggestion
const systemdScriptsPath = "/var/lib/sriov/kargs.sh"
```

Reply:
done